dev-requirements.txt

```diff
@@ -7,6 +7,7 @@ jsonpath_ng
 mock; python_version <= '3.7' # missing AsyncMock added in py38
 pylint
 pytest-asyncio
+pytest-benchmark
```

Comment:
Should this one be in this PR?

Reply:
`pytest-benchmark` was something I needed to install for the tests to run - so I figured I'd add it to the requirements. it might be that I was wrong though or misunderstoof, do you think it's not necessary?